e2e/tests/components/character-count/fixtures.ts

```diff
@@ -0,0 +1,10 @@
+import { expect, test as base } from '../../../fixtures'
+
+const test = base.extend({
+  getCharacterCountMessage: ({ page }, use) => {
```

Comment:
Debated whether a simple function taking a `Page` would've done here (the fixture is just a function here), but went with the fixture approach as it seemed more consistent (up for debate though).